rotkehlchen/db/snapshots.py

```diff
@@ -99,8 +125,36 @@ def export(self, timestamp: Timestamp, directory_path: Optional[Path]) -> Tuple[
         Otherwise, a zip file is created and the snapshot generated is written to the file
         and a path to the zip file is returned.
         """
-        timed_balances = self.get_timed_balances(timestamp)
-        timed_location_data = self.get_timed_location_data(timestamp)
+        main_currency = self.db.get_settings().main_currency
+        try:
+            main_currency_price = PriceHistorian.query_historical_price(
+                from_asset=A_USD,
+                to_asset=main_currency,
+                timestamp=timestamp,
+            )
+        except NoPriceForGivenTimestamp:
+            manual_prices = GlobalDBHandler.get_manual_prices(
+                from_asset=A_USD,
+                to_asset=main_currency,
+            )
+            if len(manual_prices) == 0:
+                main_currency_price = Price(FVal(1))
+                MessagesAggregator().add_error(f'''
+                    Could not find price for timestamp {timestamp}. Using USD for export.
+                    Please add manual price from USD to your main currency {main_currency}''')
```

Comment:
I believe is better to skip the timestamp than giving mixing values here since it can do weird things. Also 
```suggestion
                MessagesAggregator().add_error(
                  f'Could not find price for timestamp {timestamp}. Using USD for export. '
                  f'Please add manual price from USD to your main currency {main_currency}',
                )
```

Reply:
@yabirgb  Hm... @LefterisJP  has previously said in this PR:
> What you should do if this happens is probably stay with USD and export that value, while warning the user they should add a manual price because we can't find the price at the time to convert to their chosen currency.

So better skip, right? Just asking to be sure :)